src/index.html

```diff
@@ -2,52 +2,49 @@
 <html lang="en">
 
 <head>
-	<meta charset="utf-8">
-	<title>Provincial Land Use Planning</title>
-	<base href="/">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<meta name="apple-mobile-web-app-capable" content="yes">
-	<meta name="mobile-web-app-capable" content="yes">
-	<meta name="description" content="Find, learn about, and comment on active Land Use Planning engagements in British Columbia.">
-	<link rel="icon" type="image/x-icon" href="./favicon.ico">
-    <link rel="preload" as="font" type="font/woff2" href="/assets/fonts/MaterialIconsOutlined-Regular.woff2" crossorigin="anonymous">
-
-    <meta property="og:title" content="Provincial Land Use Planning" />
-    <meta property="og:type" content="website" />
-    <meta property="og:url" content="https://landuseplanning.gov.bc.ca" />
-    <meta property="og:image" content="/assets/images/land_use_share_1200x630.jpg" />
-    <meta property="og:description" content="Get involved and learn about land use planning projects in B.C." />
-
-    <meta name="twitter:title" content="Provincial Land Use Planning">
-    <meta name="twitter:description" content="Get involved and learn about land use planning projects in B.C.">
-    <meta name="twitter:image" content="/assets/images/land_use_share_1200x630.jpg">
-    <meta name="twitter:card" content="summary_large_image">
+    <meta charset="utf-8">
+    <title>Planning in Partnership</title>
+    <base href="/">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="description"
+        content="Find, learn about, and comment on active land and water planning engagements in British Columbia.">
```

Comment:
I'm guessing this is just a default value that is overwritten by the seoService?